pkg/specs/chart.go

```diff
@@ -390,3 +376,91 @@ func (r *Resolver) maybeSplitMultidocYaml(ctx context.Context, localPath string)
 
 	return nil
 }
+
+type ListK8sYaml struct {
+	APIVersion string        `json:"apiVersion" yaml:"apiVersion"`
+	Kind       string        `json:"kind" yaml:"kind" hcl:"kind"`
+	Items      []interface{} `json:"items" yaml:"items"`
+}
+
+func (r *Resolver) maybeSplitListYaml(ctx context.Context, path string) error {
```

Comment:
Could we get a test for this function? You should be able to largely copy https://github.com/replicatedhq/ship/blob/master/pkg/specs/chart_test.go#L125 for the test code